lib/config/src/app/http.rs

```diff
@@ -0,0 +1,31 @@
+/// Defines an HTTP request.
+#[derive(
+    schemars::JsonSchema, serde::Serialize, serde::Deserialize, PartialEq, Eq, Clone, Debug,
+)]
+pub struct HttpRequest {
+    /// Path to the health check endpoint.
+    pub path: String,
+    /// HTTP method to use for the health check.
+    ///
+    /// Defaults to GET.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub method: Option<String>,
+    /// Request body.
+    pub body: Option<String>,
+    /// Timeout for the health check.
+    ///
+    /// Deaults to 120s.
+    ///
+    /// Format: 1s, 5m, 11h, ...
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub timeout: Option<String>,
+    /// Expected status codes that are considered a pass for the health check.
+    #[serde(skip_serializing_if = "Vec::is_empty")]
+    pub expected_status_codes: Vec<u16>,
+    /// Optional text that is in the body of the response that is considered a pass for the health check.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub expected_body_includes: Option<String>,
+    /// Regular expression tested against the body that is considered a pass for the health check
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub expected_body_regex: Option<String>,
```

Comment:
I'd wrap all this in an expect struct:
```
expect:
  status_codes: []
  body:
    includes:
    regex:
```

Reply:
This layout was already used for health checks.
We can change things, since health checks aren't really exposed yet, but we first need to check , can't just break the schema.

Reply:
If we were not using them, then better reason for doing any breaking change now rather than later?